src/newViews/Tagging/Sources/BulkUpload/FilesUploaded/LeadEdit/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useMemo } from 'react';
+import { connect } from 'react-redux';
+import {
+    _cs,
+    formatDateToString,
+} from '@togglecorp/fujs';
+import {
+    useForm,
+} from '@togglecorp/toggle-form';
+
+import { AppState } from '#typings';
+import {
+    activeUserSelector,
+    activeProjectIdFromStateSelector,
+} from '#redux';
+
+import { FileUploadResponse } from '../../types';
+import { schema, PartialFormType } from '../../../LeadEditModal/LeadEditForm/schema';
+import LeadEditForm from '../../../LeadEditModal/LeadEditForm';
+
+import styles from './styles.scss';
+
+const mapStateToProps = (state: AppState) => ({
+    activeUser: activeUserSelector(state),
+    activeProject: activeProjectIdFromStateSelector(state),
+});
+
+interface Props {
+    className?: string;
+    activeProject: number;
+    activeUser: { userId: number };
+    file?: FileUploadResponse;
+}
+
+function LeadEdit(props: Props) {
+    const {
+        className,
+        activeProject,
+        file,
+        activeUser: {
+            userId,
+        },
+    } = props;
+
+    const partialLead: PartialFormType = useMemo(() => ({
+        title: file?.title,
+        project: activeProject,
+        assignee: userId,
+        sourceType: file?.sourceType ?? 'website',
+        publishedOn: formatDateToString(new Date(), 'yyyy-MM-dd'),
+        confidentiality: 'unprotected',
+        isAssessmentLead: false,
+        priority: 100,
+        attachment: {
+            id: file?.id,
+            file: file?.file,
+            mimeType: file?.mimeType,
+            title: file?.title,
+        },
```

Comment:
Why not just pass file?

Reply:
IDK if attachment should be an object in PartialForm

Reply:
It should not be a part of PartialForm.
I've removed it from PartialForm behavior in another PR.